csrankings.csv

```diff
@@ -9208,8 +9209,8 @@ Joseph Hummel,University of Illinois at Chicago,https://www.cs.uic.edu/~jhummel,
 Joseph Izraelevitz,University of Colorado Boulder,https://www.colorado.edu/faculty/izraelevitz,CtQUnFUAAAAJ
 Joseph J. Boutros,Texas A&M at Qatar,http://www.josephboutros.org,lyjl3nMAAAAJ
 Joseph J. DiStefano III,University of California - Los Angeles,http://www.cs.ucla.edu/joseph-distefano-iii,NOSCHOLARPAGE
-Joseph J. LaViola,University of Central Florida,http://www.eecs.ucf.edu/~jjl,kXZaMu8AAAAJ
-Joseph J. LaViola Jr.,University of Central Florida,http://www.eecs.ucf.edu/~jjl,kXZaMu8AAAAJ
+Joseph T. Kider Jr.,University of Central Florida,https://www.ist.ucf.edu/Contact/StaffDirectory/JosephKider.aspx,uQxgKlwAAAAJ
+Joseph J. LaViola Jr.,University of Central Florida,https://www.cs.ucf.edu/~jjl/,kXZaMu8AAAAJ

```

Comment:
And here, aliases needed: https://dblp.uni-trier.de/pers/l/LaViola_Jr=:Joseph_J=.html